src/ISA/ISA.Spreadsheet/ArcAssay.fs

```diff
@@ -0,0 +1,88 @@
+﻿module ISA.Spreadsheet.ArcAssay
+
+open ISA
+open FsSpreadsheet
+
+let obsoloteAssaysLabel = "ASSAY METADATA"
```

Comment:
Shoould this be written up modular? Like is "METADATA" used somewhere else too? How about "PERFORMERS"? 

If so, then this should just be `assaysLabel` + "metadataLabel"